docs/reference/cat/nodes.asciidoc

```diff
@@ -48,6 +48,9 @@ Valid columns are:
 `heap.percent`, `hp`, `heapPercent`::
 (Default) Maximum configured heap, such as `7`.
 
+`heap.max`, `hm`, `heapMax`::
+(Default) Total heap, such as `4gb`.
+
```

Comment:
Thanks for bringing up this issue @vincetrumental. I realize that we don't document `heap.max`, so I added it as part of your PR ➕